documentation/airport-file-standards.md

```diff
@@ -13,7 +13,7 @@ If you have not contributed before, please note that we will need to add you to
 
 ## Standard Requirements
 ### Terrain GeoJSON File
-Even if it is empty, a terrain file must be included as part of the pull request. We ask that the qGIS files used in the generation of the terrain GeoJSON file also be uploaded to the [openscope/other-files](https://github.com/openscope/other-files) repository. This way, if an airspace boundary is changed, we can regenerate the terrain file in far less time. This takes a long time to generate, and is quite a pain, but there are people on slack who would be happy to do it for you. See the Reference Materials for a video demonstrating an example of the only way we've currently figured out to generate terrain. If you happen to know anybody who is handy with GIS software, we could sure use some help figuring out how to improve this process, because it is _definitely_ not the way the "pros" would do it.
+Even if it is empty, a terrain file must be included as part of the pull request. We ask that the qGIS files used in the generation of the terrain GeoJSON file also be uploaded to the [openscope/other-files](https://github.com/openscope/other-files) repository. This way, if an airspace boundary is changed, we can regenerate the terrain file in far less time. This takes a long time to generate, and is quite a pain, but there are people on slack who would be happy to do it for you. See the Reference Materials for a step-by-step guide on how to generate terrain usin qGIS.
```

Comment:
`how to generate terrain usin qGIS`

yeehaw! usin that thar qGIS! 🤠

Reply:
also the repository should be `airport-modeling` instead of other-files, which I should probably actually delete anyway...? 🤔